### DISCUSSION TITLE
Creating a CCE workload/container

### DISCUSSION THREAD

**Discussion by marcosdiasdev** [huaweicloud/terraform-provider-huaweicloud#1155] [Q&A] [answered]
Is it possible to create a CCE container with the Terraform provider? The docs only cover cluster and node creation, but not workloads creation.

**Comment by niuzhenguo** [comment, marked as answer]
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs
Should use kubernetes provider to create workload/container resources.

**Reply by marcosdiasdev** [reply]
Thank you!
